### PR TITLE
 Initial commit (move code from did-io).

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

**Comment by dmitrizagidulin** [comment]
@dlongley or @mattcollier - can you enable Travis on this repo, when you have a chance?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  "description": "A library for managing cryptographic keys using Linked Data.",
+  "homepage": "http://github.com/digitalbazaar/crypto-ld",
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "http://digitalbazaar.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/digitalbazaar/crypto-ld"
```
Comment (mattcollier) [inline package.json]: Oh the irony! `https`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    "DID",
+    "Credential"
+  ],
+  "scripts": {
+    "mocha": "nyc mocha ./tests/*.spec.js",
+    "test": "npm run mocha",
+    "preversion": "npm test",
+    "postversion": "git push --follow-tags"
+  },
+  "eslintConfig": {
```
Comment (mattcollier) [inline package.json]: Use `.eslintrc` files

**Comment by davidlehn** [comment]
travis enabled

File: tests/ld-key-pair.spec.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const chai = require('chai');
+chai.use(require('dirty-chai'));
+chai.should();
```
Comment (mattcollier) [inline tests/ld-key-pair.spec.js]: I see that the `should` api is not actually being used.  So maybe it should be removed? If it stays, it's good to define globals in the eslintrc file in the test folder like:

https://github.com/digitalbazaar/bedrock-ledger-node/blob/master/test/mocha/.eslintrc

The `assertNoError` global there is a bedrock thing, so should not be included here.

Comment (dmitrizagidulin) [reply tests/ld-key-pair.spec.js]: Ah, for some reason, I thought `expect` also required a `should()` invocation. Removed it.

**Comment by dmitrizagidulin** [comment]
@mattcollier any other obstacles, or can we merge this?

**Pull request merged**